samples/angular-todo/src/app/features/journey/form/form.component.ts

```diff
@@ -161,7 +199,17 @@ export class FormComponent implements OnInit {
    */
   handleStep(step?: FRStep) {
     this.step = step;
-
+    const selectIdCallback = step.callbacks.find((cb) => cb.payload.type === 'SelectIdPCallback');
+    const callBackInputValue = selectIdCallback?.payload.input[0].value as string;
```

Comment:
Since "callback" is one word, can we keep the case of the word consistent throughout our code. Right now, I see `callback` and `callBack`. I'd prefer the first: `callback`, and if it's the second word in the name: `Callback`, like `myCallback`.

Reply:
done